CRM/Contact/Import/Parser.php

```diff
@@ -706,27 +706,21 @@ public static function exportCSV($fileName, $header, $data) {
    * @param int $id
    * @param array $params
    */
-  public function updateImportRecord($id, &$params) {
+  public function updateImportRecord(int $id, array $params): void {
     $statusFieldName = $this->_statusFieldName;
     $primaryKeyName = $this->_primaryKeyName;
 
     if ($statusFieldName && $primaryKeyName) {
-      $dao = new CRM_Core_DAO();
-      $db = $dao->getDatabaseConnection();
-
       $query = "UPDATE $this->_tableName
                       SET    $statusFieldName = ?,
                              ${statusFieldName}Msg = ?
                       WHERE  $primaryKeyName = ?";
-      $args = [
-        $params[$statusFieldName],
-        CRM_Utils_Array::value("${statusFieldName}Msg", $params),
-        $id,
-      ];
-
-      //print "Running query: $query<br/>With arguments: ".$params[$statusFieldName].", ".$params["${statusFieldName}Msg"].", $id<br/>";
 
-      $db->query($query, $args);
+      CRM_Core_DAO::executeQuery($query,[
```

Comment:
Style cop doesn't like this @eileenmcnaughton `Expected one space after the comma, 0 found`